contrib/factory-config-vpn

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh -e
+
+## This is an OnChanged handler that can handle wireguard VPN config
+## files created by fioctl.
+
+delete_vpn() {
+	if nmcli connection | grep -q factory-vpn ; then
```

Comment:
maybe grep for "^factory-vpn" to make sure we don't grep some other similar connection by mistake?